sql-ballerina/Module.md

```diff
@@ -0,0 +1,302 @@
+## Overview
+
+This module provides the common interface and functionality to interact with a database. The corresponding database
+clients can be created by using specific database modules such as `MySQL` or using the Java Database Connectivity 
+module `JDBC`.
+
+### List of Database Modules
+Ballerina now has [JDBC module](https://ballerina.io/learn/api-docs/ballerina/#/jdbc) as the generic db connector module to connect with any relational database by simply providing the JDBC URL and the other related properties.
+Ballerina also provides specially designed various database specific db connectors, so that you can work with different databases, and you are allowed to access their db specific functionalities.
+
+### Client
+
+The database client should be created using any of the above-listed database modules and once it is created, the operations and functionality explained below can be used. 
+
+#### Connection Pool Handling
+
+All database modules share the same connection pooling concept and there are 3 possible scenarios for 
+connection pool handling.  For its properties and possible values, see the `sql:ConnectionPool`.  
```

Comment:
```suggestion
connection pool handling.  For its properties and possible values, see the `sql:ConnectionPool`.  
```
Should we add the link to refer?

Reply:
URL link is added in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/82683f32ef4007d69a6e1c1608661144f35d5ac7